.travis.yml

```diff
@@ -9,10 +9,16 @@ before_install:
 before_script:
   - curl https://pre-commit.com/install-local.py | python -
   - pre-commit install
+addons:
+  - sonarcloud:
+    organization: "rgposadas"
+    token:
+      secure: $SONAR_TOKEN
```

Comment:
Is this the correct name of our token?

Reply:
Yes, cause the token is a private environment variable saved in our Travis build.